### PR TITLE
Literally all the JSON edited

### PR DESCRIPTION
**Pull request by MgreytakUW**
James, please just make sure I didn't massacre your boy.

image

### CODE REVIEW

File: Chapters/Chapter 5/ch_5_2_dialogue.json
Comment (deckerous) [inline Chapters/Chapter 5/ch_5_2_dialogue.json:1]: twiddling lol

Comment (MgreytakUW) [reply Chapters/Chapter 5/ch_5_2_dialogue.json:1]: Lol... you can edit or ignore if you hate it

File: Chapters/Chapter 7/ch7_1_dialogue.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
-        "text": "Seems like open mic night is bringing in some out of this world musicians!"
+        "text": "Open Mic Night is bringing in some out of this world musicians!"
       },
       {
         "character": "You",
-        "text": "Yeah, I noticed...that bassist-"
+        "text": "Yeah, I noticed...that... bassist?"
       },
       {
         "character": "Guy",
```
Comment (deckerous) [inline Chapters/Chapter 7/ch7_1_dialogue.json:16]: this needs to be character : Cy

**Review by deckerous — APPROVED** [review]
Thank you for putting a second (much needed) set of eyes on this!

**Comment by deckerous** [comment]
my branch has some changes to the json files, will compare and resolve conflicts

**Pull request merged**